config/locales/es/admin.yml

```diff
@@ -1,4 +1,6 @@
 es:
+  true_value: 'Si'
+  false_value: 'No'
```

Comment:
I think it would be better move this keys to `shared:` in this file: (Aslo add an accent to `Sí`)

```
admin:
  (...)
  shared:
    true_value: 'Sí'
    false_value: 'No'
```

To use:
`t("admin.shared.true_value") and t("admin.shared.false_value")`

😉